### ISSUE TITLE
nested inner formset

### ISSUE THREAD

**Issue by bmeyer71** [mbertheau/jquery.django-formset#1]
I was just wondering how I would go about adding an inner formset within the existing inner formset?
What I have is like the following:
- property
  - unit
    - unit image
  - unit
    - unit image

So each property can have multiple units. And each unit can have multiple images.

**Comment by mbertheau** [comment]
Hmm, in https://github.com/mbertheau/jquery.django-formset#a-nested-bootstrap-tabbed-formset each Block can have multiple Buildings, and each Building can have multiple Tenants. Isn't that what you're looking for?

**Comment by bmeyer71** [comment]
Thank you for your reply.

I've actually been using that as my example. Using that example, how would you go about adding a formset within the Tenants/Inner formset? Tenants in your example would be at the same level as 'unit' in my example. So I need to add another formset for 'unit image'. Hope that makes sense.

**Comment by mbertheau** [comment]
Okay, I see. Basically you just continue the nesting. Where exactly are you stuck?

**Comment by bmeyer71** [comment]
There are a couple of things. I'm not sure where to place the template code for the empty form for the 'unit image' formset. I've tried just duplicating the inner formset but not sure if that is correct since it doesn't create new forms. Which leads to the jquery script. How do I get that to just create the 'unit image' formsets? I was able to get it to work partially as it would finally create new 'unit image' formsets, but then the inner form would not work anymore.
